drivers/i2c/i2c_ll_stm32_v1.c

```diff
@@ -0,0 +1,371 @@
+/*
+ * Copyright (c) 2017, I-SENSE group of ICCS
+ * Copyright (c) 2017 Linaro Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * I2C Driver for: STM32F1, STM32F2, STM32F4 and STM32L1
+ *
+ */
+
+#include <clock_control/stm32_clock_control.h>
+#include <clock_control.h>
+#include <misc/util.h>
+#include <kernel.h>
+#include <board.h>
+#include <errno.h>
+#include <i2c.h>
+#include "i2c_ll_stm32.h"
+
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_I2C_LEVEL
+#include <logging/sys_log.h>
+
+#define I2C_REQUEST_WRITE	0x00
```

Comment:
LL API does not provide LL_I2C_REQUEST_WRITE for the families supported from I2C v1 controller